### PR TITLE
Alerts notifications

### PR DESCRIPTION
**Pull request by NemanjaManot**

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -1,25 +1,17 @@
...
-    flex: 1,
-    backgroundColor: '#fff',
-    alignItems: 'center',
-    justifyContent: 'center',
-  },
-});
+export default class App extends React.Component {
+    render() {
+        return (
+            <View style={{ flex: 1, paddingTop: STATUSBAR_HEIGHT }}>
```
Comment (marijarahman) [inline App.js]: Move style to styles object

File: src/components/AlertsNotifications/AlertNotifications.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+import { Gradient } from '../../shared/';
+import { TopButtons } from './TopButtons';
+import List from './ListAlertsAndNotifications';
+import { FooterAlertsNotifications } from './FooterAlertsNotifications'
+
+import { styles } from './style';
+
+
+export default class AlertNotifications extends React.Component {
+
```
Comment (marijarahman) [inline src/components/AlertsNotifications/AlertNotifications.js]: you don't need this much breaks (above class AlertNotifications also)

File: src/components/AlertsNotifications/AlertNotifications.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    constructor(){
+        super();
+        this.state = {
+            selectedFilterAlerts: 'initial'
+        }
+    }
+
+    render() {
+        const { container, pickerContainer, pickerStyle } = styles;
```
Comment (marijarahman) [inline src/components/AlertsNotifications/AlertNotifications.js]: :+1:

File: src/components/AlertsNotifications/ListAlertsAndNotifications.js [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+                    style={touchableAlert}
+                    onPress={() => { return this.onClickShowMore(item) }}
+                >
+                    <Icon
+                        name='warning'
+                        color={colors2['redWarning']}
+                        iconStyle={{ marginLeft: 10 }}
+                    />
+
+                    <View style={{ flexDirection: 'column', marginLeft: -30 }}>
```
Comment (marijarahman) [inline src/components/AlertsNotifications/ListAlertsAndNotifications.js]: move style to styles object

File: src/components/AlertsNotifications/TopButtons.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+import { styles } from './style';
+
+export const TopButtons = () => {
+    const { topButtons, buttonsStyle, buttonText } = styles;
+
+    return (
+        <View style={topButtons}>
+            <TouchableOpacity style={[buttonsStyle, { backgroundColor: 'rgb(32, 177, 151)'} ]}>
+                <Text style={styles.buttonText}>ACTIVE</Text>
```
Comment (marijarahman) [inline src/components/AlertsNotifications/TopButtons.js]: you've destructed buttonText

File: src/components/AlertsNotifications/style.js [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+        marginRight: '5%',
+        marginTop: 20,
+        backgroundColor: colors2['footerBackgroundColor']
+    },
+    pickerStyle: {
+        color: colors2['gray']
+    },
+
+
+
```
Comment (marijarahman) [inline src/components/AlertsNotifications/style.js]: too many breaks

File: src/utils/AlertsDataHelper.js [hunk 7]
```diff
@@ -0,0 +1,92 @@
+export const data = [
```
Comment (marijarahman) [inline src/utils/AlertsDataHelper.js:1]: I like this :)

File: src/Colors.js [hunk 8]
```diff
@@ -0,0 +1,10 @@
+export const colors2 = {
+    primaryColor: 'rgb(13, 40, 50)',
+    primaryColor2: 'rgb(27,84,106)',
+    footerBackgroundColor: 'rgb(11, 41, 53)',
+    mainGreen: 'rgb(32, 177, 151)',
+    redWarning: 'rgb(156, 57, 53)',
+    white: '#fff',
```
Comment (marijarahman) [inline src/Colors.js]: is there a reason to use hex over rgb? and we should use one or another @marko2606

Comment (NemanjaManot) [reply src/Colors.js]: There is no special reason, expect if we are using rgba. I will fix that.
Hex is probably better solution, i think.

File: src/components/AlertsNotifications/AlertNotifications.js [hunk 9]
```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { View, Picker } from 'react-native';
+
+import { Gradient } from '../../shared/';
+import { TopButtons } from './TopButtons';
+import List from './ListAlertsAndNotifications';
```
Comment (marko2606) [inline src/components/AlertsNotifications/AlertNotifications.js]: Ne znam koliko je dobra praksa da naziv importa bude drugaciji od naziva fajla koji se importuje? To bi mogli isto da prodiskutujemo...

Comment (NemanjaManot) [reply src/components/AlertsNotifications/AlertNotifications.js]: Slazem se

File: src/components/AlertsNotifications/AlertNotifications.js [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+                    </View>
+
+                    {/* --- Alerts and Notifications list --- */}
+                    <List />
+
+                </View>
+
+                {/* --- Footer --- */}
+                <FooterAlertsNotifications />
+
```
Comment (marko2606) [inline src/components/AlertsNotifications/AlertNotifications.js]: Nepotrebni komentari, moze da se vidi iz naziva komponente sta je u pitanju.

File: src/components/AlertsNotifications/ListAlertsAndNotifications.js [hunk 11]
```diff
@@ -0,0 +1,143 @@
...
+                    }}
+                    keyExtractor={(item) => { return item.key }}
+                    extraData={this.state}
+                    ListFooterComponent={() => {
+                        return this.renderFooter()
+                    }}
+                    onEndReachedThreshold={1}
+                    onEndReached={({ distanceFromEnd }) => {
+                       // console.log('on end reached ', distanceFromEnd);
+                    }}
```
Comment (marko2606) [inline src/components/AlertsNotifications/ListAlertsAndNotifications.js]: Ostao iskomentarisan kod

File: src/components/AlertsNotifications/ListAlertsAndNotifications.js [hunk 12]
```diff
@@ -0,0 +1,143 @@
...
+                        name='warning'
+                        color={colors2['redWarning']}
+                        iconStyle={{ marginLeft: 10 }}
+                    />
+
+                    <View style={{ flexDirection: 'column', marginLeft: -30 }}>
+                        <Text style={{ color: 'white' }}>
+                            {item.name}
+                        </Text>
+                        <Text style={{ color: 'rgb(146, 164, 170)', fontSize: 10 }}>SPEED OVER 15% INSTRUCTED</Text>
```
Comment (marko2606) [inline src/components/AlertsNotifications/ListAlertsAndNotifications.js]: I ovi stilovi u style objekat

File: src/components/AlertsNotifications/ListAlertsAndNotifications.js [hunk 13]
```diff
@@ -0,0 +1,143 @@
...
+                            <Icon
+                                name="keyboard-arrow-down"
+                                color='rgb(146, 164, 170)'
+                                iconStyle={{ marginRight: 10 }}
+                            />
+                    }
+                </TouchableOpacity>
+
+                {
+                    item.activeClick ? this.moreInfoAfterClick(item) : null
```
Comment (marko2606) [inline src/components/AlertsNotifications/ListAlertsAndNotifications.js]: Ako je item.activeClick inicijalno definisan kao boolean, onda bi trebalo da ostane boolean. Mozda bolje umesto null da stavis false?

File: src/Colors.js (hunk 8 above)
Comment (marko2606) [reply src/Colors.js]: We can check what type is being used in Stena Web App, and use that one...

**Review by marijarahman — APPROVED** [review]
Ready :+1:

**Review by marko2606 — APPROVED** [review]

**Review by marijarahman — APPROVED** [review]

**Pull request merged**